tox.ini

```diff
@@ -1,6 +1,5 @@
 [tox]
 envlist = py36
-skip_missing_interpreters = True
```

Comment:
Just curious why this was removed?

Reply:
If a user was missing Python 3.6 then the tests would pass anyway. Since we only have the one interpreter we should have it fail if it's missing